offerzen/global/employers/employers@4.1.js

```diff
@@ -388,7 +389,11 @@ window.$loaded(function (window, document, $, undefined) {
         url: '/search/skills?term=' + inputValue + '',
         contentType: 'application/json',
         success: function (skills) {
-          skillList = skills
+          skillList = skills.sort(function(a, b){
+            if(a.text < b.text) { return -1; }
+            if(a.text > b.text) { return 1; }
+            return 0;
+          })
```

Comment:
```suggestion
          skillList = skills.sort(function(a, b){
            return a.text.localeCompare(b.text)
          })
```

Reply:
Cool this looks a lot better so ill commit your suggestion @jethroflanagan